apps/web/src/styles/index.css

```
@@ -333,3 +333,256 @@
+  margin-bottom: 0;
+}
+.docs-copy-btn {
+  position: absolute;
+  top: 0.5rem;
+  right: 0.5rem;
+  padding: 0.2rem 0.6rem;
+  font-size: 0.75rem;
+  font-family: var(--font-sans);
+  font-weight: 500;
+  color: hsl(var(--muted-foreground));
+  background-color: hsl(var(--muted) / 0.5);
```

Stylelint (17.7.0)

[error] Expected "0.5" to be "50%" (alpha-value-notation)

(alpha-value-notation)